lib/blueprinter/extractors/auto_extractor.rb

```diff
@@ -4,11 +4,13 @@ def initialize
       @hash_extractor = HashExtractor.new
       @public_send_extractor = PublicSendExtractor.new
       @block_extractor = BlockExtractor.new
+      @datetime_formatter = DateTimeFormatter.new
     end
 
     def extract(field_name, object, local_options, options = {})
       extraction = extractor(object, options).extract(field_name, object, local_options, options)
-      value = options.key?(:datetime_format) ? format_datetime(extraction, options[:datetime_format]) : extraction
+      
+      value = @datetime_formatter.extract(extraction, options)
```

Comment:
In order to follow the current naming paradigm, how do you feel if we renamed `DateTimeFormatter#extract` to `DateTimeFormatter#format`? Since `#extract` makes more sense with the `Extractor` classes, but not so for this Formatter class.